### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S crates,.yarn,./yarn.lock -L crate,otification`

### CODE REVIEW

File: website/blog/2022-10-06_v0.16.mdx [hunk 1]
```diff
@@ -61,7 +61,7 @@ dependencies will be installed within the project.
 ### Routing `outDir` to the cache
 
 A requirement for using project references is that each project must compile declarations (`.d.ts`)
-so that consumers/dependees can resolve type information. While this makes sense, it becomes rather
+so that consumers/dependencies can resolve type information. While this makes sense, it becomes rather
```
Comment (milesj) [inline website/blog/2022-10-06_v0.16.mdx:64]: @kianmeng  dependees is actually correct, but maybe dependents is a better choice

**Comment by milesj** [comment]
@kianmeng Looks like you need to run prettier on these.

**Comment by milesj** [comment]
Thanks!

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**